runtimes/pythonrt/Makefile

```diff
@@ -0,0 +1,44 @@
+ifeq ($(shell uname),Linux)
+	open=xdg-open
+else
+	open=open
+endif
+
+all:
+	$(error please pick a target)
+
+ci: deps test-py
+
+test: test-py test-go
+
+test-py:
+	ruff check .
+	python -m pytest \
+		--doctest-modules \
+		--ignore testdata \
+		-v .
+test-go:
+	golangci-lint run ./...
```

Comment:
we're doing some trick in the root makefile to make sure this is installed.

Reply:
The make command will fail if golangci-lint is not installed, IMO it's OK.
Also, IMO `staticcheck` is the more popular linter (used by VSCode). What's the reason we prefer golangci-lint?

Reply:
reason is ignorance. can you change everything to it?

Reply:
Opended ENG-643 for it.